src/nostr-fundraiser/nostr-fundraiser.ts

```
@@ -0,0 +1,333 @@
+    } catch (error) {
+      if (seq !== this.progressLoadSeq) return;
+
+      console.error('[NostrFundraiser] Failed to fetch fundraiser progress:', error);
+      this.totalRaised = 0;
+      this.donorCount = 0;
+      this.percentRaised = 0;
+      this.remainingAmount = this.parsedFundraiser.targetAmountSats;
+      this.cachedZapDetails = [];
+      this.donationErrorMessage = 'Failed to load donors from relays';
+    } finally {
+      if (seq !== this.progressLoadSeq) return;
```

Biome (2.4.11)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)